File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
      if(x<0) return false;
      long long rev=0;
      long long temp=x;
      while(x>0)
      {
        long long  k=x%10;
        rev=rev*10+k;
        x=x/10;
      }  
      if(rev==temp)
      return true;
      return false;
        
    }
};

File: 9-palindrome-number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(num) {
    if(num<0)
    return false;
    let rev=0;
    let temp=num;
    while(temp>0){
        rev=rev*10+(temp%10);
        temp=Math.floor(temp/10);
    }
    if(rev==num)
        return true;
    return false;
};